rmf_site_editor/src/workcell/save.rs

```diff
@@ -265,6 +265,21 @@ pub fn save_workcell(world: &mut World) {
             },
             ExportFormat::Urdf => {
                 info!("Saving to urdf");
+                let urdf = match workcell.to_urdf_string() {
+                    Ok(urdf) => urdf,
+                    Err(err) => {
+                        error!("Save failed: {err}");
+                        continue;
+                    }
+                };
+                match std::io::Write::write_all(&mut std::io::BufWriter::new(f), urdf.as_bytes()) {
+                    Ok(()) => {
+                        info!("Save successful");
+                    }
+                    Err(err) => {
+                        error!("Save failed: {err}");
+                    }
+                }
```

Comment:
Is there a way to keep the I/O functions in `rmf_site_format` the same way the saving for other formats is done? Maybe a `workcell.to_urdf_writer(f)`?

Reply:
Done in cba2ebc.